lib/dotcom_web/components/live_components/trip_planner_results_section.ex

```diff
@@ -16,28 +16,61 @@ defmodule DotcomWeb.Components.LiveComponents.TripPlannerResultsSection do
 
   @impl true
   def render(assigns) do
+    grid_column_layout = "md:grid-cols-[1fr_1fr]"
+
+    grid_column_layout =
+      case assigns.results.result do
+        nil -> "md:grid-cols-[1fr]"
+        _ -> "md:grid-cols-[1fr_1fr]"
+      end
+
+    assigns = assign(assigns, :grid_column_layout, grid_column_layout)
+
     ~H"""
-    <section class="flex w-full border border-solid border-slate-400">
-      <div :if={@error} class="w-full p-4 text-rose-400">
-        <%= inspect(@error) %>
-      </div>
-      <.async_result :let={results} assign={@results}>
-        <div :if={results} class="w-full p-4">
-          <.itinerary_panel
-            results={results}
-            details_index={@expanded_itinerary_index}
-            target={@myself}
-          />
+    <div>
+      <section class={"flex flex-col md:grid #{@grid_column_layout} md:grid-rows-[min-content_1fr] w-full border border-solid border-slate-400"}>
+        <div :if={@error} class="w-full p-4 text-rose-400">
+          <%= inspect(@error) %>
         </div>
-      </.async_result>
-      <.live_component
-        module={MbtaMetro.Live.Map}
-        id="trip-planner-map"
-        class="h-96 w-full relative overflow-none"
-        config={@map_config}
-        pins={[@from, @to]}
-      />
-    </section>
+        <.async_result :let={results} assign={@results}>
+          <div
```

Comment:
This added some extra spacing between `View all options` and the content below it. 

I'm currently leaning towards leaving it as-is for this PR, and fixing it, as well as a bunch of other minor visual issues as a follow-up.